infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -118,6 +119,8 @@
      */
     public static final Duration SHUTDOWN_TIMER = Duration.ofSeconds(5);
 
+    private static final Duration OP_SPIKE_LATENCY = Duration.ofSeconds(1); //Consider 1 second as the spike for r/w/sync disk operation
+    private static final Duration OP_SPIKE_CNT = Duration.ofSeconds(30); //If continuously the spike happens for 30 seconds, report a disk problem.
```

Comment:
These values seem to be way off, for many databases latency greater than 30ms is really bad.